src/main/java/org/broadinstitute/hellbender/tools/sv/SVCallRecord.java

```diff
@@ -0,0 +1,217 @@
+
+    public static SVCallRecord create(final VariantContext variant) {
+        Utils.nonNull(variant);
+        Utils.validate(variant.getAttributes().keySet().containsAll(nonDepthCallerAttributes), "Call is missing attributes");
+        final String startContig = variant.getContig();
+        final int start = variant.getStart();
+        final String endContig = variant.getAttributeAsString(GATKSVVCFConstants.END_CONTIG_ATTRIBUTE, "NA");
+        final int end = variant.getAttributeAsInt(VCFConstants.END_KEY, variant.getStart());
+        final StructuralVariantType type = variant.getStructuralVariantType();
+        final List<String> algorithms = variant.getAttributeAsStringList(GATKSVVCFConstants.ALGORITHMS_ATTRIBUTE, GATKSVVCFConstants.DEPTH_ALGORITHM);
+        final String strands = variant.getAttributeAsString(GATKSVVCFConstants.STRANDS_ATTRIBUTE, "0");
+        if (strands.length() != 2) {
+            throw new IllegalArgumentException("Strands field is not 2 characters long");
+        }
+        final String startStrandChar = strands.substring(0, 1);
+        if (!startStrandChar.equals(SVCallRecordCodec.STRAND_PLUS) && !startStrandChar.equals(SVCallRecordCodec.STRAND_MINUS)) {
+            throw new IllegalArgumentException("Valid start strand not found");
+        }
+        final String endStrandChar = strands.substring(1, 2);
+        if (!endStrandChar.equals(SVCallRecordCodec.STRAND_PLUS) && !endStrandChar.equals(SVCallRecordCodec.STRAND_MINUS)) {
+            throw new IllegalArgumentException("Valid end strand not found");
+        }
+        final boolean startStrand = startStrandChar.equals(SVCallRecordCodec.STRAND_PLUS);
+        final boolean endStrand = endStrandChar.equals(SVCallRecordCodec.STRAND_PLUS);
+        final int length = variant.getAttributeAsInt(GATKSVVCFConstants.SVLEN, 0);
+        return new SVCallRecord(startContig, start, startStrand, endContig, end, endStrand, type, length, algorithms, variant.getGenotypes());
+    }
+
+    public static SVCallRecord createDepthOnlyFromGCNV(final VariantContext variant, final double minQuality) {
+        Utils.nonNull(variant);
+        final List<Genotype> passing = variant.getGenotypes().stream()
+                .filter(Genotype::isCalled)
+                .filter(g -> Integer.valueOf((String)g.getExtendedAttribute(GermlineCNVSegmentVariantComposer.QS)) >= minQuality)
+                .collect(Collectors.toList());
+        if (passing.isEmpty()) return null;
+        final List<String> algorithms = Collections.singletonList(GATKSVVCFConstants.DEPTH_ALGORITHM);
+
+        //TODO : use new vcfs to get actual allele
+        final int copyNumber = Integer.valueOf((String)variant.getGenotypes().get(0).getExtendedAttribute(GermlineCNVSegmentVariantComposer.CN));
+        if (copyNumber == 2) return null;
+        final boolean isDel = copyNumber < 2;
+        final boolean startStrand = isDel ? true : false;
+        final boolean endStrand = isDel ? false : true;
```

Comment:
Now that I'm looking at this again, I don't understand this logic.  Inversions, sure, but dupes?